src/site/content/en/blog/multi-screen-window-placement/index.md

```diff
@@ -0,0 +1,439 @@
+(see
+[Issue&nbsp;28](https://github.com/webscreens/window-placement/issues/28)
+for details).
+
+### New fullscreen options
+
+Up until now, you could request elements be displayed in fullscreen mode via the aptly named
+[`requestFullScreen()`](https://developer.mozilla.org/en-US/docs/Web/API/Element/requestFullscreen)
+method. The method takes an `options` parameter where you can pass
+[`FullscreenOptions`](https://developer.mozilla.org/en-US/docs/Web/API/FullscreenOptions). Up until
+now, the only option was
+[`navigationUI`](https://developer.mozilla.org/en-US/docs/Web/API/FullscreenOptions/navigationUI).
+The Multi-Screen Window Placement API adds a new `screen` property that allows you to determine
+which screen to start the fullscreen view on, for example, the primary screen:
+
+```js
+try {
+  const primaryScreen = (await getScreens()).filter((screen) => screen.primary)[0];
+  await document.body.requestFullscreen({ screen: primaryScreen });
+} catch (err) {
+  console.error(err.name, err.message);
+}
+```
+
+### Polyfill
+
+It is not possible to polyfill the functionality of the Multi-Screen Window Placement API, but you
+can shim its shape so you can code exclusively against the new API:
+
+```js
+if (!("getScreens" in window)) {
+  // Returning a one-element array with the current screen,
+  // noting that there might be more.
+  window.getScreens = async () => [window.screen];
+  // Returning `false`, noting that this might be a lie.
+  window.isMultiScreen = async () => false;
+}
+```
+
+The other aspects of the API (the `onscreenschange` event and the `screen` property of the
+`FullscreenOptions`) would simply never fire or silently be ignored respectively by non-supporting
+browsers.
+
+## Demo
+
+If you are anything like me, you keep a close eye on the development of the various cryptocurrencies
+(in reality I very much do not, but, for the sake of this article, just assume I did). In order to
+keep track of the cryptocurrencies that I own, I have developed a web app that allows me to watch
+the markets in all life situations, for example, from the comfort of my bed, where I have a decent
+single-screen setup.
+
+<figure class="w-figure">
+  <img src="./tv.jpg" alt="Massive TV screen at the end of a bed with the author's legs partly visible. On the screen a fake crypto currency trading desk. ">
+  <figcaption class="w-figcaption">Relaxing and watching the markets.</figcaption>
+</figure>
+
+This being about crypto, the markets can get hectic at any time. Should this happen, I can quickly
+move over to my desk where I have the said multi-screen setup. I can click on any currency's window
+and quickly see the full details in a fullscreen view on the opposite screen. Below is a recent
+photo of me taken during the last [YCY bloodbath](https://www.youtube.com/watch?v=dQw4w9WgXcQ)
+that caught me off-guard and
+[with hands on my face](https://www.buzzfeednews.com/article/gavon/brokers-with-hands-on-their-faces).
```

Comment:
The grammar goes off the rails here. (Dangling preposition.) I not sure what you want to say, but I'd start by starting a new sentence with "it really caught  me off guard, and..."

Reply:
I have split the sentence. The meme here is brokers with faces on their hands… https://www.buzzfeednews.com/article/gavon/brokers-with-hands-on-their-faces